### ISSUE TITLE
Web visualizations

### ISSUE THREAD

**Issue by A-Guldborg** [WilliamHeidemann/GitInsight#81]
Prerequisite: #79 is merged into main

Implement visualizations to your .Net Blazor front-ends that look similar to those in the illustrations on top of this page. These visualizations should present the results of the two analyses that you already implemented in your GitInsight applications, i.e., the commit frequencies over time and the and the commit frequencies over time per author. You might want to use bar charts or another suitable chart type to present the analyses results.

The data will come from #80 but can be developed via dummy data for now.

**Comment by A-Guldborg** [comment]
Relates to #83
